test-harness/specs/dynamic_json_responses_additionalProperties.oas3.txt

```diff
@@ -0,0 +1,31 @@
+====test====
+Prism can generate dynamic responses when the `additionalProperties` schema is set to false
+====spec====
+openapi: 3.0.0
+paths:
+  /resources/{resource_id}:
+    get:
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Resource'
+components:
+  schemas:
+    Resource:
+      type: object
+      properties:
+        name:
+          type: string
+        tags:
+          type: string
+      additionalProperties: false
```

Comment:
@XVincentX, would it be feasible to add:
1. a tests with `additionalProperties: true`
2. a test with `additionalProperties` as an obj
?

Reply:
Done. It didn't really change the final output though.